lib/features/thread/presentation/thread_controller.dart

```diff
@@ -1064,4 +1064,13 @@ class ThreadController extends BaseController with EmailActionController {
       );
     }
   }
+
+  Future<bool> swipeEmailAction(BuildContext context, PresentationEmail email, DismissDirection direction) async {
+    if (direction == DismissDirection.startToEnd) {
+      ReadActions readActions = !email.hasRead ? ReadActions.markAsRead : ReadActions.markAsUnread;
+      
+      markAsEmailRead(email, readActions);
+    }
+    return false;
```

Comment:
Why always return false? What happens if return true?

Reply:
Cause true means the item will be disappeared from the list and false means it will be moved back to its original location.